README.md

```diff
@@ -314,16 +314,16 @@ In all cases, the original request and headers are logged for the SRE Team to re
 
 ## Deploy on OpenShift
 
-The required parameters can be injected as environment variables through a secret. Create the `exhort-secret` Secret before deploying the application.
+The required parameters can be injected as environment variables through a secret. Create the `dependency-analytics-secret` Secret before deploying the application.
 
 ```bash
-oc create secret generic -n exhort --from-literal=api-trustify-token=<api_token> exhort-secret
+oc create secret generic -n exhort --from-literal=api-trustify-token=<api_token> dependency-analytics-secret
 ```
 
-After that you can use the [exhort.yaml](./deploy/exhort.yaml)
+After that you can use the [dependency-analytics.yaml](./deploy/dependency-analytics.yaml)
 
 ```bash
-oc apply -f deploy/exhort.yaml
+oc apply -f deploy/dependency-analytics.yaml
```

Comment:
rename it to trust-da.yaml and don't forget to update the references in the file to use the `trust-da-secret`